java/vector/src/main/java/org/apache/arrow/vector/validate/ValidateVectorBufferVisitor.java

```diff
@@ -283,6 +287,7 @@ public Void visit(NullVector vector, Void value) {
   }
 
   @Override
+  @SuppressWarnings("VoidUsed")
```

Comment:
You could use `vector.getUnderlyingVector().accept(this, null);` and get rid of the suppressed warning.